pkg/controller/persistentvolume/persistentvolume_controller.go

```diff
@@ -73,6 +73,30 @@ const annBoundByController = "pv.kubernetes.io/bound-by-controller"
 // provisioning to choose a particular storage class (aka profile).
 const annClass = "volume.alpha.kubernetes.io/storage-class"
 
+// This annotation is added to a PV that has been dynamically provisioned by
+// Kubernetes. It's value is name of volume plugin that created the volume.
```

Comment:
to be clear, I don't think the volume plugin name is appropriate or helpful for end users.  I would rather see this as something a bit more abstract...